enterprise/cmd/frontend/internal/auth/azureoauth/session.go

```diff
@@ -31,6 +36,13 @@ func (s *sessionIssuerHelper) GetOrCreateUser(ctx context.Context, token *oauth2
 		return nil, "failed to read Azure DevOps Profile from oauth2 callback request", errors.Wrap(err, "azureoauth.GetOrCreateUser: failed to read user from context of callback request")
 	}
 
+	if allow, err := s.verifyAllowOrgs(ctx, token); err != nil {
+		return nil, "error in verifying authorized user organizations", err
+	} else if !allow {
+		msg := "user is not part of authorized orgs list"
```

Comment:
We have a different [error message for Github](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/enterprise/cmd/frontend/internal/auth/githuboauth/session.go?L73), let's make them sound similar:
```suggestion
		msg := "user does not belong to allowed organizations"
```